configs/gatsbyjs.json

```diff
@@ -20,10 +20,37 @@
       "tags": [
         "blog"
       ]
+    },
+    {
+      "url": "https://www.gatsbyjs.com/docs/",
+      "tags": [
+        "docs"
+      ]
+    },
+    {
+      "url": "https://www.gatsbyjs.com/cloud/docs/",
+      "tags": [
+        "docs"
```

Comment:
Unsure if this should be a separate tag? (actually not sure _how_ this works). Feedback welcome!

Reply:
Should be good this way